scripts/zones/Ilrusi_Atoll/mobs/Carrion_Crab.lua

```diff
@@ -3,21 +3,16 @@
 --  Mob: Carrion Crab
 -----------------------------------
 local ID = require("scripts/zones/Ilrusi_Atoll/IDs")
+local zoneUtil = require("scripts/zones/Ilrusi_Atoll/globals/zoneUtil")
+require("scripts/globals/missions")
 -----------------------------------
 local entity = {}
 
-entity.onMobDeath = function(mob, player, isKiller)
-end
-
-entity.onMobDespawn = function(mob)
+entity.onMobDeath = function(mob, player, isKiller, firstCall)
```

Comment:
I think firstCall may still need to be handled.  Looking at current declaration, looks like this parameter would receive noKiller:

`auto result = onMobDeath(LuaMobEntity, optLuaAllyEntity, isKiller, noKiller);`

Reply:
`if firstCall then` can be changed to `if isKiller or noKiller then` to get the same result. Think I saw a `firsCall` typo btw.